package.json

```diff
@@ -87,7 +87,7 @@
     "extract-zip": "2.0.1",
     "fs-extra": "11.1.0",
     "get-pixels": "3.3.3",
-    "gridplus-sdk": "2.4.3",
+    "gridplus-sdk": "2.5.0",
```

Comment:
need to review the release notes and make sure to test any Lattice changes thoroughly here to prevent anything breaking about the way we use the SDK, I know we've had issues in the past